drivers/sensor/mhz19b/mhz19b.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_MHZ19B_MHZ19B
+#define ZEPHYR_DRIVERS_SENSOR_MHZ19B_MHZ19B
+
+#include <zephyr.h>
+#include <device.h>
+#include <init.h>
```

Comment:
Check if the rest of the headers are necessary here, init also seems to be needed only in the source file. Otherwise lgtm.